Tests/Integration/Model/CardTest.swift

```diff
@@ -50,9 +50,352 @@ class CardTest: UpholdTestCase {
+            XCTAssertEqual(transaction.refundedById, "foobiz", "Failed: Wrong transaction refundedById.")
+            XCTAssertEqual(transaction.status, "pending", "Failed: Wrong transaction status.")
+            XCTAssertEqual(transaction.type, "transfer", "Failed: Wrong transaction type.")
+            XCTAssertEqual(transactionCardDepositRequest.origin, "foobiz", "Failed: Wrong transaction card deposit origin.")
+            XCTAssertEqual(transactionCardDepositRequest.securityCode, "1234", "Failed: Wrong transaction card deposit security code.")
+
+            expectation.fulfill()
+        }
+
+        wait()
+    }
+
+    func testCreateTransactionCardDepositShouldReturnUnexpectedResponseError() {
+        let card: Card = Mapper().map("{}")!
+        card.adapter = MockRestAdapter()
+        let expectation = expectationWithDescription("Card test: create transaction transfer.")
+        let transactionDenominationRequest = TransactionDenominationRequest(amount: "foo", currency: "bar")
+        let transactionCardDepositRequest = TransactionCardDepositRequest(denomination: transactionDenominationRequest, origin: "foobar", securityCode: "1234")
+
+        card.createTransaction(transactionCardDepositRequest).error { (error: ErrorType) -> Void in
+            guard let error = error as? UnexpectedResponseError else {
+                XCTFail("Error should be UnexpectedResponseError.")
+
+                return
+            }
+
+            XCTAssertNil(error.code, "Failed: Wrong code.")
+            XCTAssertEqual(error.description, "Card id should not be nil.", "Failed: Wrong message.")
+
+            expectation.fulfill()
+        }
+
+        wait()
+    }
+
+    func testCreateTransactionCardDepositWithCommitShouldReturnTheTransaction() {
+        let card: Card = Fixtures.loadCard()
+        let expectation = expectationWithDescription("Card test: create transaction transfer.")
+        card.adapter = MockRestAdapter(body: Mapper().toJSONString(Fixtures.loadTransaction(["transactionId": "foobar"]))!)
+        let transactionDenominationRequest = TransactionDenominationRequest(amount: "foo", currency: "bar")
+        let transactionCardDepositRequest = TransactionCardDepositRequest(denomination: transactionDenominationRequest, origin: "foobar", securityCode: "1234")
+
+        card.createTransaction(true, transactionRequest: transactionCardDepositRequest).then { (transaction: Transaction) -> () in
+            XCTAssertEqual(transaction.id, "foobar", "Failed: Wrong transaction id.")
+
+            expectation.fulfill()
+        }
+
+        wait()
+    }
+
+    func testCreateTransactionDepositShouldReturnTheTransaction() {
+        let card: Card = Fixtures.loadCard()
+        let expectation = expectationWithDescription("Card test: create transaction deposit.")
+        let json: String = "{" +
+            "\"id\": \"foobar\"," +
```

Comment:
Only assert the `id` since we only need to make sure that it is a transaction that we are receiving.